### DISCUSSION TITLE
[ERROR] TypeError: iceTransport.getSelectedCandidatePair is not a function.

### DISCUSSION THREAD

**Discussion by zx310** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#2066] [General]
Test with safari brower on IOS15 and IOS16 and IOS17,I can not view the h264 video.
log shows:
[2024-10-23T06:16:03.585Z] [ERROR] TypeError: iceTransport.getSelectedCandidatePair is not a function. (In 'iceTransport.getSelectedCandidatePair()', 'iceTransport.getSelectedCandidatePair' is undefined)
![30dfb6d00e563cb7c5ac903eb5bc0b0](https://github.com/user-attachments/assets/9be1159c-e47f-471f-a82c-ee2dd39b330e)

**Comment by stefankiesz** [comment]
It is working for me on Safari on iOS 18.0.1
Can you see if the issue persists on the latest iOS version?

<img src="https://github.com/user-attachments/assets/8cad72ef-0e41-483f-83aa-ff9325a54bfe" width="300">
